src/custom-surf/pages/ServiceTicketDetail.tsx

```diff
@@ -332,15 +353,50 @@ const ServiceTicketDetail = () => {
                                                             id: "tickets.action.background_job_restarted",
                                                         })
                                                     );
-                                                    redirect("/tickets");
+                                                    redirect(`/tickets/${ticket._id}`);
                                                 })
                                             }
                                         >
                                             {intl.formatMessage({ id: "tickets.action.restart_open_relate" })}
                                         </EuiButton>
                                     </EuiFlexItem>
                                 )}
+
+                            {ticket?.transition_action === null && ticketHasUnfinishedInform(ticket) && (
+                                <EuiFlexItem grow={false} style={{ minWidth: 200 }}>
+                                    <EuiButton
+                                        color={"danger"}
+                                        iconType="refresh"
+                                        isDisabled={false}
+                                        size="m"
+                                        fill
+                                        onClick={() =>
+                                            customApiClient
```

Comment:
Could be moved to separate function. It might even be possible to use it outside component.

Reply:
Yeah this indeed turned out quite bulky, I'll move it out :+1: